pkg/workflows/testdata/fixtures/workflows/workflow_schema.json

```diff
@@ -64,6 +64,15 @@
     },
     "workflowSpecYaml": {
       "properties": {
+        "name": {
+          "type": "string",
+          "maxLength": 10,
+          "minLength": 10
+        },
+        "owner": {
+          "type": "string",
+          "pattern": "^0x[0-9a-f]{40}$"
```

Comment:
Should we allow for uppercase addresses too? They may be copy/pasting in checksummed addresses

Reply:
sure, will add.